### ISSUE TITLE
Invalid letter is used to register item in Turkish environment

### ISSUE THREAD

**Issue by Rogenar** [Kotori316/FluidTank#107]
I cannot use large fluid tank mod because fabric scala version is not updated to 1.19.3. Do you have any suggestions about this situation?

**Comment by Kotori316** [comment]
For the scala plugin, ask them to update description.

I think you're talking about [fabric-language-scala](https://www.curseforge.com/minecraft/mc-mods/fabric-language-scala).
1.19.2 and 1.19.3 are not included in the Game version, but they work fine in both version.
So you can use [Scala 1.1.0+scala.2.13.6](https://www.curseforge.com/minecraft/mc-mods/fabric-language-scala/files/3392902) for my mod.
I don't know how you install mods, but the most classical way is to put the Jar manually.

Edit: Fixed typo.

**Comment by Rogenar** [comment]
Scala 1.1.0+scala.2.13.6 version gives an error.

**Comment by Kotori316** [comment]
What kind of error did you get?
Give me crash report and minecraft logs, please.

**Comment by Rogenar** [comment]
This error is not caused by large fluid tanks mod. This is because the scala mod is out of date.

**Comment by Kotori316** [comment]
Again, what error did you get?
The scala plugin is not updated for a year(both the content and description), but I made sure that the version works in 1.19.3 environment before I upload my mod.

Tested with these files
```
InfChest-Fabric-1.19.3-19.6.jar
Jade-1.19.3-fabric-9.3.1.jar
LargeFluidTank-19.2.0-fabric.jar
RoughlyEnoughItems-10.0.592.jar
appliedenergistics2-fabric-13.0.6-beta.jar
architectury-7.1.70-fabric.jar
cloth-config-9.0.94-fabric.jar
fabric-api-0.74.0+1.19.3.jar
fabric-language-scala-1.1.0+scala.2.13.6-fat.jar
```

**Comment by Rogenar** [comment]
i tried this way and got error again
fabric-api-0.74.0+1.19.3
cloth-config-9.0.94-fabric
appliedenergistics2-fabric-13.0.6-beta
architectury-7.1.70-fabric
fabric-language-scala-1.1.0+scala.2.13.6-fat
[latest.log](https://github.com/Kotori316/FluidTank/files/10775602/latest.log)
[rei.log](https://github.com/Kotori316/FluidTank/files/10775603/rei.log)
[rei-issues.log](https://github.com/Kotori316/FluidTank/files/10775604/rei-issues.log)
[crash-2023-02-19_05.50.43-client.txt](https://github.com/Kotori316/FluidTank/files/10775605/crash-2023-02-19_05.50.43-client.txt)

LargeFluidTank-19.2.0-fabric
RoughlyEnoughItems-10.0.592

**Comment by Kotori316** [comment]
The report says something in my mod caused this crash.
```
net.minecraft.class_151: Non [a-z0-9/._-] character in path of location: fluidtank:tank_ıron
```
I don't know the letter "ı".

What language do you use in Minecraft and your OS?

**Comment by Rogenar** [comment]
I use Turkish language

**Comment by Kotori316** [comment]
OK, thank you for this information.
I use `toLowerCase` in my code but the result may contain non ascii letter in some language.
"I" should converted to "i", but failed.

I'll fix my code not to depend OS locales.

**Comment by Kotori316** [comment]
Fixed in [19.4.0](https://www.curseforge.com/minecraft/mc-mods/largefluidtank-fabric/files/4402355).
Thank you for reporting.
